client/rpc/dex_client.go

```diff
@@ -240,6 +245,74 @@ func (c *HTTP) GetDepth(tradePair string) (*types.OrderBook, error) {
 	return &ob, nil
 }
 
+func (c *HTTP) GetTimelocks(address string) (types.TimeLockRecords, error) {
```

Comment:
The response is slice, but definition here is just `types.TimeLockRecords`,  I think compile will fail.